_examples/lot/lot.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Reference from: https://github.com/initialed85/golang_python_binding_research
```

Comment:
I would add a blank line between our license and the provenance of this test file.